zh/enable-tls-for-mysql-client.md

```diff
@@ -419,6 +530,55 @@ spec:
    name: ${cluster_name}
  initSql: |-
    create database app;
+ tlsClientSecretName: ${cluster_name}-tidb-initializer-client-secret
+---
+apiVersion: pingcap.com/v1alpha1
```

Comment:
Suggest separating the backup and restore CR from the tidb cluster because the TiDB service may take a while to be ready.